lab1/style.css

```diff
@@ -0,0 +1,107 @@
+body {
+  padding: 182px 100px 50px 100px;
+  text-align: justify;
+  background-color: #819A9A;
+  font-size: 100%;
+  font-family: "Comic Sans MS", arial;
+
+}
+nav {
+  position: fixed;
+  width: 100%;
+  right: 100%;
+  left: 0;
+  top: 0;
+  z-index: 1;
+  padding: 20px 0;
+  background-color: #D7ECEC;
+  border-radius: 5px;
+  text-align: center;
+
+}
+
+a {
+  text-align: center;
+}
+
+a:link {
+    color: #000000;
+}
+
+a:visited {
+    color: #000000;
+}
+
+a:hover {
+    color: #FF00FF;
+}
+
+a:active {
+    color: #0000FF;
+}
+caption {
+  margin-bottom: 15px;
+  font-size: 20px;
```

Comment:
You can be fancy and use `em` as a unit instead here. Or even `rem` if you wanna go hipster all the way.